motd/all-in-one

```diff
@@ -3,12 +3,16 @@ Welcome to the Kayobe Lab!
 
 You may want to change your password immediately.
 
+To bootstrap Kayobe all-in-one overcloud without Magnum, run:
+
+    ./all-in-one.sh
+
 To bootstrap Kayobe all-in-one overcloud with Magnum, run:
 
     ./all-in-one-with-magnum.sh
 
 Once this is complete, use the OpenStack environment as follows:
 
     source ~/adminrc.sh
-    openstack coe cluster list
+    openstack coe cluster list # If Magnum was deployed
```

Comment:
Could we make this a new section? e.g.

If Magnum was deployed:

    openstack coe cluster list